Makefile

```diff
@@ -31,40 +32,59 @@ all: docker
 gobuild:
 	./scripts/build false
 
+# create output directories
+.out-stamp:
+	mkdir -p ./out/test-artifacts ./out/cni-plugins
+	touch .out-stamp
+
+out: .out-stamp
+
 # Basic go build
 static:
 	./scripts/build
 
+BUILDER_IMAGE="amazon/amazon-ecs-agent-build:make"
+.builder-image-stamp: scripts/dockerfiles/Dockerfile.build
```

Comment:
This is not a `PHONY` target on purpose?

Reply:
Same reasoning as `.out-stamp`